src/pages/my-courses/MyCourses.tsx

```diff
@@ -95,6 +95,45 @@ const MyCourses = () => {
     })
   }
 
+  const duplicateCourse = useCallback(
+    (id?: string) => {
```

Comment:
should id be not required?

Reply:
Yes, because the service needs "id" as string or undefined